docs/03-computation/index.mdx

```diff
@@ -6,13 +6,61 @@ tags: ["introduction"]
 import DocCardList from '@theme/DocCardList';
 import {useCurrentSidebarCategory} from '@docusaurus/theme-common';
 
-The blindnet devkit **Privacy Computation Engine** is a service that interpret your rights to hold and treat a particular Data Capture at a particular point of time and calculate a response to [Data Subjects'](/docs/references/lexicon#data-subject) [Privacy Requests](/docs/references/lexicon#privacy-request).
+The blindnet devkit **Privacy Computation Engine** (PCE) is the core of your "privacy stack".
 
-It associates:
+It is a service delivering Restful APIs to manage the two core features of the DevKit by:
 
-- a [Privacy Compiler](/docs/references/high-level-architecture#privacy-compiler)
-- and [Customization API](/docs/references/high-level-architecture#customization-api), for you to set up your organization-specific information and parameters
+1. interpreting your rights to hold and treat a particular **[Data Capture](/docs/references/lexicon#data-capture)** at a particular point in time
+2. calculating a response to [Data Subjects'](/docs/references/lexicon#data-subject) **[Privacy Requests](/docs/references/lexicon#privacy-request)**.
 
----
+Both these features are provided by a main internal component we call the [Privacy Compiler](/docs/references/high-level-architecture#privacy-compiler), based on the specific information and parameters of your organization you define using the [Customization API](/docs/references/high-level-architecture#customization-api).
+
+## How It Works
+
+The Privacy Compiler inside the Privacy Computation Engine is here to deliver recommended actions based on a timeline of _Events_[^1] it proceeds.
+
+:::info key point
+
+The Privacy Compiler ***compiles*** a timeline of ***events*** into recommended actions.
```

Comment:
Thank you for the feedback @m4rk055. I'm not sure what I should do then tho. Do you think this simplified summary is too confusing, oversimplified, or wrong?

Reply:
To be more precise, IMHO:

1. _«  Actions (Action terms in Demand) are always chosen by a user. »_ is why I always make sure to talk about ***"recommended"*** actions
2. even when talking about Data Capture, I'd expect the PCE to _"recommend"_ storing the data, i.e. send an instruction the Data Access Component CAN use to operate actions in a storage service via a Storage Connector - just on top of my mind, I'd say an alternative use case could also be to just hold the data temporarily in the Data Access Component so Data Consumers can decide what to do about each Data Capture (export and do what they want with it) ... this would indeed be a pretty bad use case security-wise, but just for the sake of example :wink:

Reply:
I'd avoid using "**recommended** actions" because of the _Action Terms_ in the domain. It might be confusing (well, it confused me).

I think this summary is enough:

PCE recommends
- **responses** to Demands by compiling data subject's event timeline
- creating data archive for ACCESS and deleting data for DELETE requests and
- in case of timeline change, deleting data having data categories outside the timeline's privacy scope

Reply:
I'd still like to have a very short sentence expressing what each component does explicitly.

What would you think about the following:

```suggestion
The Privacy Compiler ***compiles*** a timeline of ***events*** into recommended operations.
```